schema/definitions/0.8.0/schema/fmu_results.json

```diff
@@ -1054,6 +1055,21 @@
                         ]
                     }
                 }
+            },
+            "context": {
+                "description": "The internal FMU context in which this data object was produced",
+                "type": "object",
+                "required": [
+                    "stage"
+                ],
+                "properties": {
+                    "stage": {
+                        "type": "string",
+                        "examples": [
+                            "case", "iteration", "realization"
```

Comment:
Would it make sense to have this as an enum instead?

Reply:
Yes, I think so - when we have standardized its content.